lib/scryptenc/scryptenc.c

```diff
@@ -114,7 +114,7 @@ pickparams(size_t maxmem, double maxmemfrac, double maxtime,
 		return (SCRYPT_ELIMIT);
 
 	/* Figure out how fast the CPU is. */
-	if ((rc = scryptenc_cpuperf(&opps)) != 0)
+	if ((rc = scryptenc_cpuperf(&opps)) != SCRYPT_OK)
```

Comment:
Since SCRYPT_OK is defined to 0, this may seem pointless.  However, I'd like to make this change because it emphasizes that `rc` is a SCRYPT_* value.  (A few weeks ago, I spent a bit of time puzzling over `scryptenc.c` before I re-read that `rc` wasn't a generic "0 for success, -1 for failure" in this case.